R/pkg/DESCRIPTION

```diff
@@ -36,3 +36,4 @@ Collate:
     'stats.R'
     'types.R'
     'utils.R'
+RoxygenNote: 5.0.1
```

Comment:
I think this might have been auto-generated by roxygen - Can we revert this file for this PR ?